plugins/org.jkiss.dbeaver.ext.databricks/plugin.xml

```diff
@@ -31,7 +31,29 @@
 
                     <file type="jar" path="https://databricks-bi-artifacts.s3.us-east-2.amazonaws.com/simbaspark-drivers/jdbc/2.6.22/SimbaSparkJDBC42-2.6.22.1040.zip" bundle="!drivers.databricks"/>
                     <file type="jar" path="drivers/databricks" bundle="drivers.databricks"/>
-                    <file type="license" path="drivers/databricks/EULA.txt" bundle="drivers.databricks"/>/>
+                    <file type="license" path="drivers/databricks/EULA.txt" bundle="drivers.databricks"/>
+
+                    <parameter name="supports-indexes" value="false"/>
+                    <parameter name="supports-multi-insert" value="true"/>
+                    <property name="@dbeaver-default-resultset.maxrows.sql" value="true"/>
+                </driver>
+                <driver
+                        id="databricks-native"
+                        label="Azure Databricks (Native)"
+                        icon="platform:/plugin/org.jkiss.dbeaver.ext.generic/icons/databricks_icon.png"
+                        iconBig="platform:/plugin/org.jkiss.dbeaver.ext.generic/icons/databricks_icon_big.png"
+                        class="com.databricks.client.jdbc.Driver"
+                        webURL="https://www.databricks.com/spark/jdbc-drivers-download"
+                        description="Azure Databricks driver"
+                        categories="sql,analytic,cloud">
+                    <replace provider="generic" driver="azure_databricks"/>
+
+                    <file type="jar" path="maven:/com.databricks:databricks-jdbc:RELEASE[2.6.32]"
+                          bundle="!drivers.databricks"/>
+                    <file type="jar" path="drivers/databricks" bundle="drivers.databricks"/>
```

Comment:
path="drivers/databricks"

We have different jar files there. Please remove this string.